.ci-operator.yaml

```diff
@@ -1,4 +1,4 @@
 build_root_image:
   name: release
   namespace: openshift
-  tag: rhel-8-release-golang-1.16-openshift-4.10
+  tag: rhel-8-release-golang-1.17-openshift-4.11
```

Comment:
just FYI this seems to be about the [testing buildroot configured here](https://github.com/openshift/ocp-build-data/blob/06b78ebb25b078b97c862d41158b57567265a5f7/streams.yml#L42-L49). not the [etcd builder configured here](https://github.com/openshift/ocp-build-data/blob/06b78ebb25b078b97c862d41158b57567265a5f7/streams.yml#L54-L60) (still 1.16)